requirements/env_climada.yml

```diff
@@ -1,60 +1,45 @@
-name: climada_env
+name: rasterio
 channels:
+  - conda-forge
```

Comment:
Adding `conda-forge` to the `channels` list means in practice that almost all packages are installed from `conda-forge` instead of the default channel. Is this our intention? There is only a single package that requires `conda-forge`, namely `cfgrib` and that does not require this line since it comes with an explicit `conda-forge::` prefix.

Reply:
Agreed. But the channels configuration remains strange to me, for without explicit `conda-forge::` prefix, the cfgrib package cannot even be found.
```suggestion
```